testplan/web_ui/testing/package.json

```diff
@@ -2,60 +2,65 @@
   "name": "testplan",
   "version": "0.2.0",
   "private": true,
-  "resolutions": {
-    "@babel/core": "7.13.15",
-    "@babel/preset-env": "7.13.15"
+  "pnpm": {
+    "overrides": {
+      "react": "^17.0.2",
+      "@types/react": "^17.0.2"
```

Comment:
shall we remove them?

Reply:
They are there as i tried to come over of these warnings:

```
 WARN  Issues with peer dependencies found

.
├─┬ @material-ui/core
│ ├── ✕ unmet peer @types/react@"^16.8.6 || ^17.0.0": found 18.2.6
│ ├─┬ @material-ui/styles
│ │ └── ✕ unmet peer @types/react@"^16.8.6 || ^17.0.0": found 18.2.6
│ └─┬ @material-ui/system
│   └── ✕ unmet peer @types/react@"^16.8.6 || ^17.0.0": found 18.2.6
├─┬ @material-ui/icons
│ └── ✕ unmet peer @types/react@"^16.8.6 || ^17.0.0": found 18.2.6
├─┬ @material-ui/lab
│ └── ✕ unmet peer @types/react@"^16.8.6 || ^17.0.0": found 18.2.6
├─┬ react-copy-html-to-clipboard
│ └── ✕ unmet peer react@^15.3.0: found 17.0.2
├─┬ react-vis
│ ├── ✕ unmet peer react@"15.3.0 - 16.x": found 17.0.2
│ └─┬ react-motion
│   └── ✕ unmet peer react@"^0.14.9 || ^15.3.0 || ^16.0.0": found 17.0.2
└─┬ react-scripts
  └── ✕ unmet peer typescript@"^3.2.1 || ^4": found 5.0.4
```  
But I failed, especially I do not understand the @material-ui/* ones. I f you know what to do here I'm all ears